Documentation/RelNotes/2.13.0.txt

```diff
@@ -38,7 +38,7 @@ UI, Workflows & Features
 
  * Deletion of a branch "foo/bar" could remove .git/refs/heads/foo
    once there no longer is any other branch whose name begins with
-   "foo/", but we didn't do so so far.  Now we do.
+   "foo/", but we didn't do so far.  Now we do.
```

Comment:
I would argue that this was correctly written in the original. Perhaps we could change this to "but we didn't do _that_ so far" but I'm also okay with the original wording. Dunno.

Reply:
Good catch! Thank you @Denton-L :-) 

You're surely right about the correctness of the original wording, 
and I think what you suggested is good, too, and that helps make it
no longer suspicious.